src/dotnet/Chat.UI.Blazor/Components/ChatView/Items/ChatMessageMarkup.razor

```diff
@@ -20,6 +26,9 @@
     [Inject] private IActivePlaybackInfo ActivePlaybackInfo { get; init; } = null!;
     private ChatEntry Entry => Model.Entry;
     private Markup? Markup { get; set; } = null!;
+    private bool HasTextOnly => Markup?.Parts.All(x => x is PlainTextPart) ?? false;
+    private bool IsMouseOver { get; set; }
+    private bool ShowMarkupText => HasTextOnly && !IsMouseOver;
```

Comment:
It's a bit more complicated: it should be shows like this also when it's playing. I am pretty sure nothing is highlighted now during the playback due to this.